### ISSUE TITLE
Latex math mode support in "Text" option

### ISSUE THREAD

**Issue by MrF83** [Circuit2TikZ/CircuiTikZ-Designer#49]
"Label" option of the elements supports preview of latex math mode. When typing text in the "Text" options of for example "Rectangle" element and using latex math mode ("$<math>$") preview renders dollar signs instead of math mode. When using Tiktz code in latex it renders correctly.

**Comment by JPWiedemann** [comment]
Ideally, this is exactly what I thought the feature should behave like anyway. However, I haven't really been able to consistently tell mathjax which parts of the text should and shouldn't be rendered via math mode, so for now, text will always be plain text and the label will always be rendered in math mode. You can always trick mathjax into rendering text via \text in the label. I know that this is suboptimal but it should work as a workaround

**Comment by MrF83** [comment]
Oh, okay. I really have no experience with Node.js and only a little with MathJax, but did you have a look at the [inlineMath](https://docs.mathjax.org/en/v1.1-latest/options/tex2jax.html) preprocessor option? I don't know if this would be an option.

**Comment by JPWiedemann** [comment]
Hi @MrF83, so I've implemented basic typesetting for the text component text. We can now use $-signs to mark text as math mode and it will be typeset (mostly) correct. I have tested it a bit and it's not identical to LaTeX but pretty close in most cases. This is not a general LaTeX compiler for the text, only mathjax commands in the math mode regions work. Outside math mode, text is just treated as normal text without looking for commands

**Comment by MrF83** [comment]
Very nice! Sorry for being inactive that long. I love using the designer and until now that were really the only minor issues experienced. Thanks!
